api/tasks.go

```diff
@@ -909,6 +909,9 @@ type TaskCSIPluginConfig struct {
 	// Ideally this should be the FQDN of the plugin.
 	ID string `mapstructure:"id"`
 
+	// Driver is a CSI property describing the provider service
+	Driver string `mapstructure:"driver"`
```

Comment:
Isn't this the same as `ID`?

Reply:
It's intended to be the same as CSI spec `SP` or `Storage Provider` (and if reintroduced should be called that). That field is out of this pr.